templates/config.yaml

```diff
@@ -339,103 +339,69 @@ templates:
           path: components/healthchecker
           since: "1.7"
           optional: false
-      - to: ../prow/jobs/incubator/compass/components/provisioner/provisioner-generic.yaml
+      - to: ../prow/jobs/incubator/compass/components/schema-migrator/schema-migrator-generic.yaml
         values:
           <<: *compass_generic_component
-          path: components/provisioner
+          path: components/schema-migrator
           since: "1.7"
           optional: false
-      - to: ../prow/jobs/control-plane/components/provisioner/provisioner-kcp-generic.yaml
+      - to: ../prow/jobs/incubator/compass/tests/connector-tests/connector-tests-generic.yaml
         values:
-          <<: *control_plane_generic_component
-          path: components/provisioner
+          <<: *compass_generic_component
+          path: tests/connector-tests
           since: "1.7"
           optional: false
-      - to: ../prow/jobs/incubator/compass/components/kyma-environment-broker/kyma-environment-broker-generic.yaml
+      - to: ../prow/jobs/incubator/compass/tests/director/director-generic-approach.yaml
         values:
           <<: *compass_generic_component
-          path: components/kyma-environment-broker
-          since: "1.9"
-      - to: ../prow/jobs/control-plane/components/kyma-environment-broker/kyma-environment-broker-kcp-generic.yaml
-        values:
-          <<: *control_plane_generic_component
-          path: components/kyma-environment-broker
-          since: "1.9"
-      - to: ../prow/jobs/incubator/compass/components/schema-migrator/schema-migrator-generic.yaml
+          path: tests/director
+          since: "1.10"
+      - to: ../prow/jobs/incubator/compass/tests/connectivity-adapter/connectivity-adapter-tests-generic.yaml
         values:
           <<: *compass_generic_component
-          path: components/schema-migrator
+          path: tests/connectivity-adapter
+          since: "1.11"
+          optional: false
+      - to: ../prow/jobs/control-plane/components/provisioner/provisioner-generic.yaml
+        values:
+          <<: *control_plane_generic_component
+          path: components/provisioner
           since: "1.7"
           optional: false
+      - to: ../prow/jobs/control-plane/components/kyma-environment-broker/kyma-environment-broker-generic.yaml
+        values:
+          <<: *control_plane_generic_component
+          path: components/kyma-environment-broker
+          since: "1.9"
       - to: ../prow/jobs/control-plane/components/schema-migrator/schema-migrator-kcp-generic.yaml
```

Comment:
remove prefix `kcp-`

Reply:
I cannot because we have 2 jobs `schema-migration` one in Compass and the second in kcp.

Reply:
Oh, sorry, you're right, I forgot about it. Then it looks good to me :)